flight/targets/bl/common/main.c

```diff
@@ -596,6 +607,9 @@ static void process_packet_rx(struct bl_fsm_context * context, const struct bl_m
 		bl_fsm_inject_event(context, BL_EVENT_JUMP_TO_APP);
 		break;
 	case BL_MSG_RESET:
+		PIOS_IRQ_Disable();
```

Comment:
Yes.

Re this and the previous comment-- note that this stuff preceded some of my other changes to the enumeration machinery, and may not be strictly necessary.  But it improved behavior with the old stuff and probably helps in some cases still.